blankly/exchanges/interfaces/paper_trade/metrics.py

```diff
@@ -20,6 +20,23 @@
 from blankly.utils.time_builder import build_year
 
 
+def periods_per_year(period : int) -> float:
+    """
+    Find how many trading periods occur within a trading year
+
+    Args:
+        period: the number of seconds in each trading period
+    Returns:
+        number of trading periods in each year
+        
+    """
+    # TODO: update logic to account for strategies that do not hold overnight
```

Comment:
For now, I think this is perfectly fine. This is a really valuable function especially if they put in a different resolution

Reply:
This is difficult to detect on our end and we could theoretically update this in the case of the interval that they want to do (either based on preference). Also, we should probably also detect if it's on a crypto or 24/7 exchange so it's 365 instead of 252. @EmersonDove , thoughts on how we should do this? Perhaps passing in an additional parameter of `days`?